Library/Formula/twister.rb

```diff
@@ -0,0 +1,56 @@
+require "formula"
+
+class Twister < Formula
+  homepage "http://twister.net.co"
+  url "https://github.com/miguelfreitas/twister-core/archive/v0.9.28.tar.gz"
+  sha1 "53b03636ae4d7539002fe9c3cf2dd58e9657ac1e"
+
+  head do
+    url "https://github.com/miguelfreitas/twister-core.git"
+  end
+
+  depends_on "boost" => :build
+  depends_on "miniupnpc" => :build
+  depends_on "openssl" => :build
+  depends_on "berkeley-db4" => :build
```

Comment:
There is no way ` miniupnpc `, ` berkeley-db4` and ` openssl ` at least are just build time deps here. No way.